index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" 
+    content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="styles/index.css" >
+
+    <link rel="stylesheet"
+    href="https://fonts.googleapis.com/css?family=Roboto|Crete Round|Poppins">
+    <title>Portfolio</title>
+</head>
+<body>
+  <header id="header">
+    <a>My Logo</a>  
+    
+    <button id="menu-btn"></button>
```

Comment:
- [ ] As suggested by the previous reviewer, kindly consider adding this code `<button id="menu-btn"></button>` inside `<nav id="header-nav">` before ul tag, he was asking this change because the hamburger button is part of the navigation.